src/snac/server/database/DBUtil.php

```diff
@@ -4239,9 +4288,9 @@ private function savePlace($vhInfo, $cObj, $relatedTable, $fkID)
                                                    $gObj->getID(),
                                                    $this->db->boolToPg($gObj->getConfirmed()),
                                                    $gObj->getOriginal(),
-                                                   $this->thingID($gObj->getGeoTerm()),
-                                                   $this->thingID($gObj->getType()),
-                                                   $this->thingID($gObj->getRole()),
+                                                   $this->termID($gObj->getGeoTerm()),
```

Comment:
Does GeoTerm work the same way as Term?  They're slightly different structures, so we might want to keep thingID here?

Reply:
Geoterm has some differences. I wrote an isEmpty() function for it to mirror Term, but I agree it's safer and less coupled just to use thingID().